[BE] @GeneratedValue를 IDENTITY로 수정했을 때 테스트 격리를 시킨다.

## 기능 상세
<img width="1412" alt="image" src="https://user-images.githubusercontent.com/57135043/177499860-7784194a-7e69-4ec0-8bba-2aae91d3e903.png">
- 테스트 격리가 이루어지지 않아 에러가 발생한다.

```java
Caused by: org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Unique index or primary key violation: "PUBLIC.UK_MBMCQELTY0FBRVXP1Q58DN57T_INDEX_8 ON PUBLIC.MEMBER(EMAIL NULLS FIRST) VALUES ( /* 3 */ 'email@email.com' )"; SQL statement:
insert into member (member_id, email, password, username) values (default, ?, ?, ?) [23505-214]
```

--- a/backend/src/test/java/com/woowacourse/naepyeon/acceptance/AcceptanceTest.java
+++ b/backend/src/test/java/com/woowacourse/naepyeon/acceptance/AcceptanceTest.java
@@ -6,10 +6,11 @@ import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabas
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.web.server.LocalServerPort;
 import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
+@AutoConfigureTestDatabase
 public class AcceptanceTest {
 
     @LocalServerPort

